### PR TITLE
docs: fix build and install doc to use test build flag not dev

### PR DESCRIPTION
**Pull request by themarwhal**
Signed-off-by: Marie Bremner <marwhal@fb.com>

<!--
    Tag your PR title with the components that it touches along with
    the type of change
    E.g. "fix(orc8r): Fix reindexer race condition" or "feat(agw) ..."
-->

## Summary
Follow up from https://github.com/magma/magma/issues/8322

`dev` -> produces debug AGW package
`test` -> produces non-debug AGW package

Will file another PR to add a `release` tag so we can migrate from `test`.
<!-- Enumerate changes you made and why you made them -->

## Test Plan
```
➜  docs git:(fix-agw-doc-round-2) ✗ make precommit
make -C readmes precommit
docker build -t magma_readmes .
[+] Building 4.4s (8/8) FINISHED
 => [internal] load build definition from Dockerfile                                                                                                                                                                                                     0.0s
 => => transferring dockerfile: 36B                                                                                                                                                                                                                      0.0s
 => [internal] load .dockerignore                                                                                                                                                                                                                        0.0s
 => => transferring context: 2B                                                                                                                                                                                                                          0.0s
 => [internal] load metadata for docker.io/library/node:14                                                                                                                                                                                               4.3s
 => [1/4] FROM docker.io/library/node:14@sha256:cd98882c1093f758d09cf6821dc8f96b241073b38e8ed294ca1f9e484743858f                                                                                                                                         0.0s
 => CACHED [2/4] RUN mkdir -p /readmes                                                                                                                                                                                                                   0.0s
 => CACHED [3/4] WORKDIR /readmes                                                                                                                                                                                                                        0.0s
 => CACHED [4/4] RUN npm install --global markdownlint-cli                                                                                                                                                                                               0.0s
 => exporting to image                                                                                                                                                                                                                                   0.0s
 => => exporting layers                                                                                                                                                                                                                                  0.0s
 => => writing image sha256:4f88f810fb275a5ef12ddf38b3fd309cbcec3e7a81b94f9447b9f32de4bda40a                                                                                                                                                             0.0s
 => => naming to docker.io/library/magma_readmes                                                                                                                                                                                                         0.0s
docker-compose run readmes markdownlint --ignore proposals/p010_vendor_neutral_dp.md . && echo PASSED
Creating readmes_readmes_run ... done
PASSED
```
<!--
    How did you test your change? How do you know it works?
    Add supporting screenshots, terminal pastes, etc. as necessary
-->

## Additional Information

- [ ] This change is backwards-breaking

<!--
    If this is a backwards-breaking change, document the upgrade instructions.
    All upgrade instructions for backwards-breaking changes will be aggregated
    in the next release's changelog so this is very important to fill out.
-->

### CODE REVIEW

File: docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ git checkout v1.1
 
 2. **Install prerequisites**. Make sure you have installed all the tools specified in the prerequisites https://magma.github.io/magma/docs/basics/prerequisites#prerequisites
 
-3. **Build and create deb package**. In your local magma repo, go to the path `magma/lte/gateway` and run the command `fab dev package:vcs=git`
+3. **Build and create deb package**. In your local magma repo, go to the path `magma/lte/gateway` and run the command `fab test package:vcs=git`
```
Comment (ulaskozat) [inline docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md]: can we include information for both fab commands and their impact?

File: docs/readmes/lte/build_install_magma_pkg_in_agw.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ hide_title: true
 
 2. **Install prerequisites**. Make sure you have installed all the tools specified in the prerequisites <https://magma.github.io/magma/docs/basics/prerequisites#prerequisites>
 
-3. **Build and create deb package**. In your local magma repo, go to the path `magma/lte/gateway` and run the command `fab dev package:vcs=git`
+3. **Build and create deb package**. In your local magma repo, go to the path `magma/lte/gateway` and run the command `fab test package:vcs=git`
```
Comment (ulaskozat) [inline docs/readmes/lte/build_install_magma_pkg_in_agw.md]: same comment here.

**Review by ulaskozat — APPROVED** [review]
LGTM

File: docs/readmes/lte/build_install_magma_pkg_in_agw.md (hunk 2 above)
Comment (themarwhal) [reply docs/readmes/lte/build_install_magma_pkg_in_agw.md]: Hey @ulaskozat! I updated the doc to include information about both of the commands.

File: docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md [hunk 3]
```diff
@@ -21,36 +21,49 @@ git checkout v1.1
...
-```
-vagrant ssh magma
-cd ~/magma-packages/
-```
-You will need only the ones that say `magma_1.1.XXX` and `magma-sctpd_1.1.XXX` (for v1.1 versions)
+    ```bash
+    vagrant ssh magma
+    cd ~/magma-packages/
+    ```
+    You will need only the ones that say `magma_1.1.XXX` and `magma-sctpd_1.1.XXX` (for v1.1 versions)
```
Comment (hcgatewood) [inline docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md:50]: Nit: newline before this one

File: docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md [hunk 4]
```diff
@@ -21,36 +21,49 @@ git checkout v1.1
...
 
-3. **Build and create deb package**. In your local magma repo, go to the path `magma/lte/gateway` and run the command `fab dev package:vcs=git`
+3. **Build and create deb package**.
+    To build an AGW package, use the script located at `$MAGMA_ROOT/lte/gateway/fabfile.py`. The commands below will create a vagrant machine, then build and create a deb package.
+
+    The following commands are to be run from `$MAGMA_ROOT/lte/gateway` on your host machine.
+    To create a package for production, run
+
+    ```bash
+    fab test package:vcs=git
```
Comment (hcgatewood) [inline docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md:31]: This is... weird that we're using "test" to create a pkg for production. Seems like a clear antipattern. But for a followup task I suppose

**Review by hcgatewood — APPROVED** [review]

File: docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md (hunk 4 above)
Comment (themarwhal) [reply docs/docusaurus/versioned_docs/version-1.5.0/lte/build_install_magma_pkg_in_agw.md:31]: Ya have a PR to rename it to 'release'

**Pull request merged**